File: Backoffice/src/app/Models/Inventory.ts
export class Inventory {
  public constructor(init?: Partial<Inventory>) {
    Object.assign(this, init);
  }

  id: string;
  productId: string;
  quantity: number;
  description: string;
  type: string;
  customerEmail: string;
  returnDate: string

}

File: Backend/Wiz/DataModels/DataModels/Inventory.cs
using DataModels;
using MongoDB.Bson.Serialization.Attributes;
namespace DataModels
{
    [BsonIgnoreExtraElements]
    public class Inventory: DataClass
    {
        [BsonIgnoreIfDefault]
        public string ProductId { get; set; }
        [BsonIgnoreIfDefault]
        public int Quantity { get; set; }
        [BsonIgnoreIfDefault]
        public string Description { get; set; }
        [BsonIgnoreIfDefault]
        public string Type { get; set; }
        [BsonIgnoreIfDefault]
        public string CustomerEmail { get; set; }
        [BsonIgnoreIfDefault]
        public string ReturnDate { get; set; }

    }
}

